apps/native-component-list/src/screens/Amplitude/TrackingOptionsSelector.tsx

```diff
@@ -0,0 +1,263 @@
+import { AmplitudeTrackingOptions, setTrackingOptionsAsync } from 'expo-analytics-amplitude';
+import React from 'react';
+import { PixelRatio, Switch, Text, View, Button, StyleSheet } from 'react-native';
+
+interface State {
+  optionsToSet: AmplitudeTrackingOptions;
+  options: AmplitudeTrackingOptions;
+  showList: boolean;
+  result?: string;
+  error?: string;
+}
+
+export default class TrackingOptionsSelector extends React.Component<object, State> {
+  readonly state: State = {
+    optionsToSet: {
+      disableAdid: true,
+      disableCarrier: true,
+      disableCity: true,
+      disableCountry: true,
+      disableDMA: true,
+      disableDeviceBrand: true,
+      disableDeviceManufacturer: true,
+      disableDeviceModel: true,
+      disableIDFV: true,
+      disableIPAddress: true,
+      disableLanguage: true,
+      disableLatLng: true,
+      disableOSName: true,
+      disableOSVersion: true,
+      disablePlatform: true,
+      disableRegion: true,
+      disableVersionName: true,
+    },
+    options: {
+      disableAdid: true,
+      disableCarrier: true,
+      disableCity: true,
+      disableCountry: true,
+      disableDMA: true,
+      disableDeviceBrand: true,
+      disableDeviceManufacturer: true,
+      disableDeviceModel: true,
+      disableIDFV: true,
+      disableIPAddress: true,
+      disableLanguage: true,
+      disableLatLng: true,
+      disableOSName: true,
+      disableOSVersion: true,
+      disablePlatform: true,
+      disableRegion: true,
+      disableVersionName: true,
+    },
+    showList: false,
+    result: undefined,
+    error: undefined,
+  };
+
+  _applyOptions = () => {
+    this.setState(state => ({ options: state.optionsToSet }));
+  };
+
+  _renderToggle = ({
+    title,
+    disabled,
+    valueName,
+    value,
+  }: {
+    title: string;
+    disabled?: boolean;
+    valueName:
+      | 'disableAdid'
+      | 'disableCarrier'
+      | 'disableCity'
+      | 'disableCountry'
+      | 'disableDMA'
+      | 'disableDeviceBrand'
+      | 'disableDeviceManufacturer'
+      | 'disableDeviceModel'
+      | 'disableIDFV'
+      | 'disableIPAddress'
+      | 'disableLanguage'
+      | 'disableLatLng'
+      | 'disableOSName'
+      | 'disableOSVersion'
+      | 'disablePlatform'
+      | 'disableRegion'
+      | 'disableVersionName';
+    value?: boolean;
+  }) => (
+    <View
+      style={{
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        paddingVertical: 5,
+        borderBottomWidth: 1.0 / PixelRatio.get(),
+        borderBottomColor: '#cccccc',
+      }}>
+      <Text style={{ flex: 1, fontSize: 16 }}>{title}</Text>
```

Comment:
Extract style to `StyleSheet.create`
```suggestion
    <View
      style={styles.toggleConatainer}>
      <Text style={styles.toggleLabel}>{title}</Text>
```